### PR TITLE
feat: defaults are only loaded when no configuration found

### LINKED ISSUE #194: Default configuration doesn't make sense
Hre's default configuration which is merged if there is no events for `issues` or `pull_requests` keys specified in user-defined config:
```
Configuration.DEFAULTS = {
  label: 'work in progress|do not merge|experimental|proof of concept',
  title: 'wip|dnm|exp|poc',
  stale: {
    message: 'There haven\'t been much activity here. This is stale. Is it still relevant? This is a friendly reminder to please resolve it. :-)'
  }
}
```

Consider PR titled "Implement CSV export". Mergable fails, because it finds "exp".

This default regexp could be improved by checking for keywords only around `^|$|\s`.

### PR DESCRIPTION
**Pull request by jusx**
## Goals
resolves #194 

## Changes
- Only set defaults when no configuration is found in the repos.
- Change the default title regex to be more precise. It makes more sense to only check for `wip` in the beginning of the title.
- Change the default label regex to only look for "work in progress" and "do not merge."

### CODE REVIEW

File: lib/configuration/lib/consts.js [hunk 1]
```diff
@@ -92,12 +92,12 @@ Settings : {{{displaySettings details.settings}}}
...
     do: 'title',
     must_exclude: {
-      regex: 'wip|dnm|exp|poc'
+      regex: '^wip'
     }
   }, {
     do: 'label',
     must_exclude: {
-      regex: 'work in progress|do not merge|experimental|proof of concept'
+      regex: 'work in progress|wip|do not merge'
```
Comment (rgtk) [inline lib/configuration/lib/consts.js]: I would throw `wip` here. Intentions fails with Issue title like `Creating an user wipes all others`. Would keep only verbose one, for demo purposes.

Comment (rgtk) [reply lib/configuration/lib/consts.js]: \cc @jusx

Comment (jusx) [reply lib/configuration/lib/consts.js]: @rgtk - This is validating on the `label` and not the `title`. A label named `wip` seem reasonable here. Thoughts?

Comment (rgtk) [reply lib/configuration/lib/consts.js]: Right. Thanks.

**Comment by jusx** [comment]
I was not done with this PR. Was going to check some things after resolving conflicts.

**Pull request merged**